src/main/java/org/fundacionjala/coding/melvi/BankOcr.java

```diff
@@ -0,0 +1,266 @@
+    public ArrayList<String> scanOcr(final String filePath) {
+        ArrayList<String> resultado = new ArrayList<>();
+
+        StringBuilder char1 = new StringBuilder();
+        StringBuilder char2 = new StringBuilder();
+        StringBuilder char3 = new StringBuilder();
+        StringBuilder char4 = new StringBuilder();
+        StringBuilder char5 = new StringBuilder();
+        StringBuilder char6 = new StringBuilder();
+        StringBuilder char7 = new StringBuilder();
+        StringBuilder char8 = new StringBuilder();
+        StringBuilder char9 = new StringBuilder();
+        BufferedReader br;
+        try {
+            br = new BufferedReader(new InputStreamReader(new FileInputStream(filePath), "UTF-8"));
+
+            String line;
+
+            while ((line = br.readLine()) != null) {
+                if ("".equals(line)) {
+                    break;
+                }
+                // process the line.
+                char1.setLength(0);
+                char2.setLength(0);
+                char3.setLength(0);
+                char4.setLength(0);
+                char5.setLength(0);
+                char6.setLength(0);
+                char7.setLength(0);
+                char8.setLength(0);
+                char9.setLength(0);
+                String number = "";
+                for (int i = 0; i < INT; i++) {
+                    if (i > 0) {
+                        line = br.readLine();
+                        if ("".equals(line)) {
+                            break;
+                        }
+                    }
+                    char1.append(line.substring(0, TRHEE));
+                    char2.append(line.substring(TRHEE, SIX));
+                    char3.append(line.substring(SIX, NINE));
+                    char4.append(line.substring(NINE, TWELVE));
+                    char5.append(line.substring(TWELVE, FIFTEEN));
+                    char6.append(line.substring(FIFTEEN, EIGHTEEN));
+                    char7.append(line.substring(EIGHTEEN, TWENTY_ONE));
+                    char8.append(line.substring(TWENTY_ONE, TWENTY_FOUR));
+                    char9.append(line.substring(TWENTY_FOUR, TWENTY_SEVEN));
+
+                }
+                number = identifyChar(char1.toString()) + identifyChar(char2.toString())
+                        + identifyChar(char3.toString()) + identifyChar(char4.toString())
+                        + identifyChar(char5.toString()) + identifyChar(char6.toString())
+                        + identifyChar(char7.toString()) + identifyChar(char8.toString())
+                        + identifyChar(char9.toString());
+
+                resultado.add(number);
+
+                line = br.readLine(); //empty separator line
+                if ("".equals(line)) {
+                    break;
+                }
+
+            }
+
+            if (br != null) {
+                br.close();
+            }
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)